cmstestsuite/web/__init__.py

```diff
@@ -212,16 +213,15 @@ def specific_info(self):
         res = "URL: %s\n" % self.url
         if self.response is not None:
             res += "\nREQUEST HEADERS\n"
-            for (key, value) in self.response.request.headers.iteritems():
+            for key, value in iteritems(self.response.request.headers()):
```

Comment:
Are the new `()` intended?

Reply:
Nope, this was an oversight, good catch!